src/canmatrix/tests/test_xls.py

```diff
@@ -0,0 +1,19 @@
+import canmatrix.xls
+import decimal
+
+
+def test_parse_value_name_collumn():
```

Comment:
Five more instances of `collumn` to fix.

Reply:
oh no